src/main/java/gov/usds/case_issues/services/UserService.java

```diff
@@ -0,0 +1,28 @@
+package gov.usds.case_issues.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import gov.usds.case_issues.db.model.OAuthUser;
+import gov.usds.case_issues.db.repositories.UserRepository;
+
+@Service
+@Transactional
+public class UserService {
+
+	@Autowired
+	private UserRepository _userRepo;
+
+	public void createUserOrUpdateLastSeen(String id, String printName) {
+        OAuthUser user = _userRepo.findByUserId(id);
+        if (user != null) {
+            user.updateLastSeen();
+            _userRepo.save(user);
```

Comment:
I think this is a case where we don't need to call `save`—it's a managed object that's changed inside a transaction, it should automatically get saved when we exit.

Reply:
Is there any harm in calling `save`. A magic "trust me this object is saved" seems confusing to me

Reply:
Well, that's a sort of interesting point. On the one hand, it's a no-op, so there's no harm doing it; on the other hand, it's a no-op, so it's potentially confusing to leave it in the code where people might look at it and assume that it was doing something. I personally lean toward removing it, because no-op code inevitably gets copy-pastaed, and then you have boilerplate everywhere that makes people think incorrect things about how the library works. (Which is also why I say "I think" above, because I have spent too much time reading Hibernate code written by people who didn't understand how Hibernate works, so I lack all courage of my convictions.)